test/tests/haskell-ghci/container.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+set -e
+
+echo "" | ghci
```

Comment:
All this is doing is ensuring that ghci exists in the image and does *something*. The "unnecessary use of double quotes" is because my bashfu is bad.